src/ServiceStack.Text/PclExport.NetStandard.cs

```diff
@@ -83,6 +84,15 @@ public NetStandardPclExport()
 #else 
             this.DirSep = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? '\\' : '/';
 #endif
+            var req = HttpWebRequest.Create("http://servicestack.net");
+            try
+            {
+                req.Headers[HttpRequestHeader.UserAgent] = "ServiceStack";
```

Comment:
Yeah I answered my own question immediately below since it uses reflection. Perhaps the workaround for the linker would be the client to add a mock call in their App:

    ((HttpWebRequest)WebRequest.Create("http://example.org")).UserAgent = "Leave me linker";